docs/multimodal.md

```diff
@@ -0,0 +1,105 @@
+* **Question answering**: We search the index using both the text and multimodal embeddings. We send both the text and the image to the LLM, and ask it to answer the question based on both kinds of sources.
+* **Citations**: The frontend displays both image sources and text sources, to help users understand how the answer was generated.
+
+## Using the feature
+
+### Prerequisites
+
+* The use of a chat completion model that supports multimodal inputs. The default model for the repository is currently `gpt-4.1-mini`, which does support multimodal inputs. The `gpt-4o-mini` technically supports multimodal inputs, but due to how image tokens are calculated, you need a much higher deployment capacity to use it effectively. Please try `gpt-4.1-mini` first, and experiment with other models later.
+
+### Deployment
+
+1. **Enable multimodal capabilities**
+
+   Set the azd environment variable to enable the multimodal feature:
+
+   ```shell
+   azd env set USE_MULTIMODAL true
+   ```
+
+2. **Provision the multimodal resources**
+
+   Either run `azd up` if you haven't run it before, or run `azd provision` to provision the multimodal resources. This will create a new Azure AI Vision account and update the Azure AI Search index to include the new image embedding field.
+
+3. **Re-index the data:**
+
+   If you have already indexed data, you will need to re-index it to include the new image embeddings.
+   We recommend creating a new Azure AI Search index to avoid conflicts with the existing index.
+
+   ```shell
+   azd env set AZURE_SEARCH_INDEX multimodal-index
+   ```
+
+   Then delete the `.md5` hash files in the data folder(s) and run the data ingestion process again to re-index the data:
+
+   Linux/Mac:
+
+   ```shell
+   ./scripts/prepdocs.sh
+   ```
+
+   Windows:
+
+   ```shell
+   .\scripts\prepdocs.ps1
+   ```
+
+4. **Try out the feature:**
+
+   ![Screenshot of app with Developer Settings open, showing multimodal settings highlighted](./images/multimodal.png)
+
+   * If you're using the sample data, try one of the sample questions about the financial documents.
+   * Check the "Thought process" tab to see how the multimodal approach was used
+   * Check the "Supporting content" tab to see the text and images that were used to generate the answer.
+   * Open "Developer settings" and try different options for "Included vector fields" and "LLM input sources" to see how they affect the results.
+
+5. **Customize the multimodal approach:**
+
+   You can customize the RAG flow approach with a few additional environment variables.
+
+   The following variables can be set to either true or false,
+   to control whether Azure AI Search will use text embeddings, image embeddings, or both:
```

Comment:
for searching right? And we also need to update the other descriptions of what the approach does with these variables

Reply:
should this all be default true for multimodal approach?